### PR TITLE
Devenv: Create script to provision many folders

### LINKED ISSUE #74190: Devenv: Provision a lot of folders
We have examples where performance or display of a lot of folders may be of interest. We should be able to provision this with a denenv script. Consider making one for nested folder and one for non-nested.

Provision script will:
 - accepts number of folders as an argument (defaults to 1,000)
 - generate that number of folders with a three dashboards in each

### PR DESCRIPTION
**Pull request by joshhunt**
Adds a devenv provisioning script to create many folders with dashboards. The number of folders and dashboards can be passed in or arguments.

Usage:

```sh
cd devenv
./setup.sh bulk-folders # creates 200 folders with 3 dashboards each
./setup.sh bulk-folders 1000 5  # creates 1000 folders with 5 dashboards each
```

Fixes https://github.com/grafana/grafana/issues/74190

### CODE REVIEW

**Comment by eledobleefe** [comment]
I've run `./setup.sh bulk-folders 33 3` on local and it works regarding folders but it just creates two dashboards per folder: `dashboard1.json` and `dashboard3.json` 🤔

File: devenv/bulk-folders/bulk-folders.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   echo "Creating folder $folderCounter"
+  folderPath="bulk-folders/Bulk Folder ${folderCounter}"
+
+  mkdir -p "$folderPath"
+
+  for (( dashCounter=1; dashCounter<="$numberOfDashboardsPerFolder"; dashCounter++ ))
+  do
+    jsonnet -o "$folderPath/dashboard${dashCounter}.json" -e "local bulkDash = import 'bulk-dashboards/bulkdash.jsonnet'; bulkDash + {  uid: 'bulk-folder-${folderCounter}-${dashCounter}',  title: 'Bulk Folder  ${folderCounter} Dashboard ${dashCounter}' }"
+
+    dashCounter=$((dashCounter+1))
```
Comment (eledobleefe) [inline devenv/bulk-folders/bulk-folders.sh:17]: I think we should delete this as it is increasing the counter again.

**Review by eledobleefe — CHANGES_REQUESTED** [review]

File: devenv/bulk-folders/bulk-folders.sh (hunk 1 above)
Comment (L-M-K-B) [reply devenv/bulk-folders/bulk-folders.sh:17]: Good catch! I removed it.

**Review by eledobleefe — APPROVED** [review]
LGTM now 🎉

**Pull request merged**